diff --git a/quakespasm/Quake/Makefile.darwin b/quakespasm/Quake/Makefile.darwin
index 71e25f3..cd39044 100644
--- a/quakespasm/Quake/Makefile.darwin
+++ b/quakespasm/Quake/Makefile.darwin
@@ -4,6 +4,7 @@
 # "make DEBUG=1" to build a debug client.
 # "make SDL_CONFIG=/path/to/sdl-config" for unusual SDL installations.
 # "make SDLNET=1" to use SDL_net (NOT recommended) instead of platform
+# "make SDL_FRAMEWORK_PATH=/path/to/Frameworks" to specify the directory containing SDL.framework
 # specific code.
 
 ### Enable/Disable codecs for streaming music support
@@ -82,7 +83,12 @@ endif
 # /Library/Frameworks/SDL.framework is available
 SDL_CFLAGS =-D_GNU_SOURCE=1 -D_THREAD_SAFE
 SDL_CFLAGS+=-DSDL_FRAMEWORK -DNO_SDL_CONFIG
-SDL_LIBS   =-Wl,-framework,SDL -Wl,-framework,Cocoa
+SDL_LIBS   =
+ifdef SDL_FRAMEWORK_PATH
+	SDL_CFLAGS +=-F$(SDL_FRAMEWORK_PATH) -framework SDL
+	SDL_LIBS   +=-Wl,-F,$(SDL_FRAMEWORK_PATH) 
+endif
+SDL_LIBS  +=-Wl,-framework,SDL -Wl,-framework,Cocoa
 
 ifeq ($(SDLNET),1)
 NET_LIBS   :=-Wl,-framework,SDL_net
